apps/web/src/pages/invites/MembersInvitePage.tsx

```diff
@@ -42,6 +42,25 @@ export function MembersInvitePage() {
     form.resetFields(['email']);
   }
 
+  async function removeMemberClick(member) {
+    await removeMember(member._id)
+      .then((result) => {
+        showNotification({
+          message: `Successful member deletion.`,
+          color: 'green',
+        });
+        refetch();
+
+        return result;
+      })
+      .catch((err) => {
+        showNotification({
+          message: err.message,
+          color: 'red',
+        });
+      });
+  }
```

Comment:
```suggestion
  async function removeMemberClick(member) {
    try {
        await removeMember(member._id)
   
        showNotification({
          message: `Successful member deletion.`,
          color: 'green',
        });
        
        refetch();
    } catch (e) {
        showNotification({
          message: err.message,
          color: 'red',
        });
      }
  }
```

A small suggestion can be refactored to use plain async await like the rest of the codebase. Wdyt?

Reply:
I agree.